src/Cms/AbstractWebTemplate.php

```diff
@@ -153,14 +175,31 @@ public function metaDescription()
     {
         $context = $this->contextObject();
 
+        $desc = null;
         if ($context instanceof HasMetatagInterface) {
-            return $context['meta_description'];
+            $desc = $context['meta_description'];
+        }
+
+        if (!$desc) {
+            $desc = $this->metaDescriptionFromConfig();
```

Comment:
Is there a reason the fallback methods are called `meta*FromConfig()`?
I think ``fallbackMeta*()` would be better.